src/style.scss

```diff
@@ -0,0 +1,136 @@
+$main__color: #060b35;
+$secondary__color: #616070;
+$button__color: #00acdc;
+$white__color: #fff;
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body,
+div,
+h2,
+p,
+a {
+  margin: 0;
```

Comment:
div and a tags don't have margins, so you don't need to reset them. box-sizing: border-box; you can apply to * selector.

Reply:
If i don't reset they in that way, code don't pass the tests

Reply:
`div` and tag `a` there are no default `margins`, so they are redundant in this list.